### PR TITLE
Added support for IInvocationOperation to the control flow graph.

### PR DESCRIPTION
**Pull request by 333fred**
Tagging @AlekseyTs @dotnet/roslyn-compiler @dotnet/analyzer-ioperation for review.

### CODE REVIEW

**Comment by 333fred** [comment]
Ping for reviews @AlekseyTs @dotnet/roslyn-compiler @dotnet/analyzer-ioperation

**Review by mavasani — APPROVED** [review]

**Comment by 333fred** [comment]
Ping for reviews @AlekseyTs @dotnet/roslyn-compiler

**Comment by 333fred** [comment]
@dotnet-bot retest windows_debug_vs-integration_prtest please

**Comment by 333fred** [comment]
@dotnet-bot retest windows_release_vs-integration_prtest please

File: src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs [hunk 1]
```diff
@@ -1717,6 +1717,44 @@ public override IOperation VisitCaughtException(ICaughtExceptionOperation operat
...
+            ImmutableArray<IArgumentOperation> arguments = operation.Arguments;
+            foreach (IArgumentOperation argument in arguments)
+            {
+                _evalStack.Push(Visit(argument.Value));
+            }
+
+            ImmutableArray<IArgumentOperation> visitedArguments;
+            if (!arguments.IsEmpty)
+            {
+                var builder = ArrayBuilder<IArgumentOperation>.GetInstance();
```
Comment (AlekseyTs) [inline src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs]: >GetInstance() [](start = 63, length = 13)

Do we know how many items we plan to add?

File: src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IConditionalAccessExpression.cs [hunk 2]
```diff
@@ -300,6 +303,7 @@ void M1(P input, int? result)
             VerifyFlowGraphAndDiagnosticsForTest<BlockSyntax>(source, expectedGraph, expectedDiagnostics);
         }
 
+        [CompilerTrait(CompilerFeature.IOperation, CompilerFeature.Dataflow)]
```
Comment (AlekseyTs) [inline src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IConditionalAccessExpression.cs]: >[CompilerTrait(CompilerFeature.IOperation, CompilerFeature.Dataflow)] [](start = 8, length = 69)

We should put this attribute on the type, if it is not there already

**Review by AlekseyTs — APPROVED** [review]
LGTM (iteration 3)

File: src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs (hunk 1 above)
Comment (333fred) [reply src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs]: Yes, I'll address this in a follow up PR. I'm creating a helper for pushing and popping from the stack we will be doing it regularly.

File: src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IConditionalAccessExpression.cs (hunk 2 above)
Comment (333fred) [reply src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IConditionalAccessExpression.cs]: I'll address this in a follow up PR.

Comment (agocke) [reply src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IConditionalAccessExpression.cs]: Why include `CompilerFeature.IOperation` on the method at all? Why not just put it on the type and be done with it?

**Review by agocke — COMMENTED** [review]
LGTM

**Pull request merged**